cmd/image-autobumper/main.go

```diff
@@ -308,7 +308,7 @@ func updateReferences(imageBumperCli imageBumper, filterRegexp *regexp.Regexp, o
 		return nil
 	}
 	updateYAMLFile := func(name string) error {
-		if (strings.HasSuffix(name, ".yaml") || strings.HasSuffix(name, ".yml")) && !isUnderPath(name, o.ExcludedConfigPaths) {
+		if (strings.HasSuffix(name, ".yaml") || strings.HasSuffix(name, ".yml") || strings.HasSuffix(name, ".tf")) && !isUnderPath(name, o.ExcludedConfigPaths) {
```

Comment:
```suggestion
		if (strings.HasSuffix(name, ".yaml") || strings.HasSuffix(name, ".yml") || strings.HasSuffix(name, ".tf")) || strings.HasSuffix(name, ".tfvars")) && !isUnderPath(name, o.ExcludedConfigPaths) {
```

We need to detect .tfvars files too. They contain variables values, thus may contain an image urls. We already have a .tfvars files on extraFiles list.